### PR TITLE
I made changes (obviously)

### PR DESCRIPTION
**Pull request by Poolitzer**
Did you mean [Schmeckels](https://rickandmorty.fandom.com/wiki/Schmeckle) and wrote the wrong name?

### CODE REVIEW

**Comment by PXNX** [comment]
Good one. Please accept.

File: _pages/sign-up.md [hunk 1]
```diff
@@ -19,20 +19,20 @@ You can copy the entry of `john_does_gh_username` and adjust it to your needs. M
...
-  # To add a profile picture, add it as /assets/profile_pictures/your_gh_username or add a link
+  # To add a profile picture, add it as /assets/profile_pictures/your_gh_username.png or add a link
   avatar      : "/assets/profile_pictures/john_does_gh_username.png"
   links:
     # Any relevant social links. The only mandatory one is your GitHub profile.
     - label: "GitHub"
       # fontawesome icons are optional
       icon: "fab fa-fw fa-github"
-      url: "https://github.com/tgdevbotlist/tgdevbotlist.github
+      url: "https://github.com/tgdevbotlist/tgdevbotlist.github"
```
Comment (Bibo-Joshi) [inline _pages/sign-up.md]:
```suggestion
      url: "https://github.com/tgdevbotlist/tgdevbotlist.github.io"
```
if we're at it …

File: _pages/faq.md [hunk 2]
```diff
@@ -18,16 +18,16 @@ No.
...
 # Which Jekyll template does this page use?
 
-[Minimal Mistakes](https://mademistakes.com/work/minimal-mistakes-jekyll-theme/)
+[Minimal Mistakes](https://mademistakes.com/work/minimal-mistakes-jekyll-theme/).
 
 # How many Möbius strips can you fit in a three dimensional space?
 
-This may come as a surprise, but you can only fit countably many Möbius strips in a three dimensional space, even though Möbius strips are two dimensional objects. For all the juice details, head [here](https://doi.org/10.1142/S0218216518420051).
\ No newline at end of file
+This may come as a surprise, but you can only fit countably many Möbius strips in a three dimensional space, even though Möbius strips are two dimensional objects. For all the juice details, head [here](https://doi.org/10.1142/S0218216518420051).
```
Comment (Bibo-Joshi) [inline _pages/faq.md:33]: what canged here?

File: _pages/about.md [hunk 3]
```diff
@@ -24,4 +24,4 @@ This page started in mid 2021 and we'll have to see how well it will be received
 * Spend a lot of resources on advanced search features or a high-end website - if you want that, please head to some commercial page
 * Offer legal advice for anything - just some general [rules of thumb](/advice/)
 * Take responsibility for any information provided by the developers
-* Track any of your data - this page is hosted on [GitHub Pages](https://pages.github.com), so obviously GitHub has access to your IP and such but apart from that this page doesn't use any analytics service
\ No newline at end of file
+* Track any of your data - this page is hosted on [GitHub Pages](https://pages.github.com), so obviously GitHub has access to your IP and such but apart from that this page doesn't use any analytics service
```
Comment (Bibo-Joshi) [inline _pages/about.md:27]: what changed here?

**Review by Bibo-Joshi — CHANGES_REQUESTED** [review]
> Did you mean Schmeckels and wrote the wrong name?

No, just wrote something that came to my mind when thinking of exotic currencies. Apparently it's a real one - Israel. But probably it's better to use a made up one.

File: _pages/about.md (hunk 3 above)
Comment (Poolitzer) [reply _pages/about.md:27]: nothing. i didnt change anything.

File: _pages/faq.md (hunk 2 above)
Comment (Poolitzer) [reply _pages/faq.md:33]: again, nothing

Comment (Bibo-Joshi) [reply _pages/faq.md:33]: mh, maybe newline-thingies …

**Pull request merged**